### PR TITLE
Minimum versions for Ignition dependencies

### PR DESCRIPTION
**Pull request by chapulina**
The main motivation is that right now, if a user only calls `sudo apt install libignition-gazebo2` to upgrade only this package, they end up with a broken system because the dependencies are not up-to-date. The result is a runtime error when running `ign gazebo` and it can't load the library.

I based myself on the versions currently required on [ign-gazebo's CMakeLists](https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo2/CMakeLists.txt), but we haven't been very good about updating these, so some could be out of date.

Also see https://github.com/ignitionrobotics/ign-gazebo/pull/148.

### CODE REVIEW

File: ubuntu/debian/control [hunk 1]
```diff
@@ -18,13 +18,13 @@ Build-Depends: cmake,
                libignition-gui2-dev,
```
Comment (j-rivero) [inline ubuntu/debian/control]:
```suggestion
               libignition-gui2-dev (>= 2.1.0),
```

File: ubuntu/debian/control [hunk 2]
```diff
@@ -18,13 +18,13 @@ Build-Depends: cmake,
...
                libignition-math6-dev,
```
Comment (j-rivero) [inline ubuntu/debian/control]:
```suggestion
               libignition-math6-dev (>= 6.2.0),
```

File: ubuntu/debian/control [hunk 3]
```diff
@@ -18,13 +18,13 @@ Build-Depends: cmake,
...
                libignition-math6-eigen3-dev,
```
Comment (j-rivero) [inline ubuntu/debian/control]:
```suggestion
               libignition-math6-eigen3-dev (>= 6.2.0),
```

**Review by j-rivero — CHANGES_REQUESTED** [review]
some build-dep need sync with the rest. Looks good to merge once those are updated.

**Comment by chapulina** [comment]
> some build-dep need sync with the rest

Thanks, done!

**Review by j-rivero — APPROVED** [review]

**Pull request merged**